pkg/sdk/accounts.go

```diff
@@ -397,9 +487,6 @@ func (opts *DropAccountOptions) validate() error {
 	if !ValidObjectIdentifier(opts.name) {
 		errs = append(errs, ErrInvalidObjectIdentifier)
 	}
-	if !validateIntGreaterThanOrEqual(opts.gracePeriodInDays, 3) {
```

Comment:
Because we agreed not to validate Snowflake hardcoded values in SDK. It will be validated on the resource level though. Remember that some hardcoded values (e.g. upper limit for this value) can be adjusted depending on a given customer needs.